fctrl2/fctrl2.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <vector>
+#include <cstring>
+
+using namespace std;
+
+void printVector(vector<int> v) {
+    vector<int>::reverse_iterator it;
+    for(it = v.rbegin(); it != v.rend(); ++it) {
```

Comment:
explain why you are printing in reverse order (hint: you are saving in reverse order)